plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/hms/HiveMetastoreTableOperations.java

```diff
@@ -97,7 +99,15 @@ protected void commitToExistingTable(TableMetadata base, TableMetadata metadata)
             }
         }
         finally {
-            thriftMetastore.releaseTableLock(lockId);
+            try {
+                thriftMetastore.releaseTableLock(lockId);
+            }
+            catch (Exception e) {
+                log.warn("Release lock step has failed for lockId: \"%s\". Not throwing the exception while releasing the lock, after commit has already succeeded. " +
+                        "So, that underlying iceberg API will not do the metadata cleanup, otherwise table will be in unusable state." +
+                        "If configured and supported, the unreleased lock will be automatically released by the metastore after not hearing a heartbeat for a while, " +
+                        "or otherwise it might need to be manually deleted from the metastore backend storage. exception: \"%s\"", lockId, e);
```

Comment:
put `e` as the first argument, so the whole stacktrace gets printed

Reply:
updated